### ISSUE TITLE
Erro Codigo de Barras ITAU

### ISSUE THREAD

**Issue by rafahcardoso** [QuilhaSoft/JasperPHP-OpenBoleto#2]
Bom dia.
Gerei um boleto pela biblioteca que me gerou um codigo de barras. Porem o nosso numero que passo para o banco no arquivo remessa é 0, o banco que cria.
Com isso, o código de barras do boleto gerado com o codigo de barras com o boleto gerado pelo banco ( ja com o nosso numero ) fica diferente.

Alguem ja passou por isso?

Codigo de barras gerado pela biblioteca:
34191.12002 00000.840678 22845.440001 7 77240000040000

Gerado pelo banco:
34191.12804 68586.790674 22845.440001 8 77240000040000

**Comment by Rctnet** [comment]
Bom dia, normalmente as carteiras em que o banco gera o nosso numero também são onde eles geram o boleto, nesse caso você deve obter o nosso numero do banco primeiro para depois gerar o boleto a ser impresso, já com o nosso numero do banco.

**Comment by rafahcardoso** [comment]
Entao, mas o cliente quer que ele ja imprima o boleto com a nota e no final da tarde envie o arquivo de remessa para o banco.

E se eu passo no arquivo remessa o nosso numero ele da erro pois fala q eu preciso passar 000000.

Tem que mudar alguma coisa no banco sera??


Enviado do Outlook<http://aka.ms/weboutlook>

________________________________
De: Rogerio Muniz <notifications@github.com>
Enviado: quinta-feira, 8 de novembro de 2018 09:33
Para: QuilhaSoft/JasperPHP-OpenBoleto
Cc: rafahcardoso; Author
Assunto: Re: [QuilhaSoft/JasperPHP-OpenBoleto] Erro Codigo de Barras ITAU (#2)


Bom dia, normalmente as carteiras em que o banco gera o nosso numero também são onde eles geram o boleto, nesse caso você deve obter o nosso numero do banco primeiro para depois gerar o boleto a ser impresso, já com o nosso numero do banco.

**Comment by Rctnet** [comment]
Sim, tem que mudar a carteira... para uma que o nosso numero seja só seu. habitualmente chamam de boleto com emissão pelo beneficiário.
